projects/1.19/assets/new-mutant-more/mutantmore/lang/zh_cn.json

```
@@ -0,0 +1,422 @@
+{
+	"entity.mutantmore.concoction_w" : "混合物W",
+	"entity.mutantmore.mutation_cloud" : "突变云雾",
+	"entity.mutantmore.wither_slash" : "凋灵刃气",
+	"entity.mutantmore.mutant_shulker_bullet" : "突变潜影贝导弹",
+	"entity.mutantmore.custom_shulker_bullet" : "潜影贝导弹",
+	"entity.mutantmore.mutant_blaze_fireball" : "突变烈焰人火球",
+	"entity.mutantmore.mutant_wither_skeleton" : "突变凋灵骷髅",
+	"entity.mutantmore.mutant_shulker" : "突变潜影贝",
+	"entity.mutantmore.mutant_blaze" : "突变烈焰人",
+	"entity.mutantmore.mutant_husk" : "突变尸壳",
+	"entity.mutantmore.mutant_hoglin" : "突变疣猪兽",
+	"entity.mutantmore.mutant_phantom" : "突变幻翼",
+	"entity.mutantmore.rodling" : "烈焰灵",
+	"entity.mutantmore.rodling_fireball" : "烈焰灵火球",
+	"entity.mutantmore.thrown_mutation_potion" : "突变药水",
+	"entity.mutantmore.wither_bomb" : "凋灵爆弹",
+	"entity.mutantmore.mutant_wither_skeleton_body_part" : "突变凋灵骷髅残肢",
+	"entity.mutantmore.mutant_shulker_turret" : "突变潜影贝炮塔",
+	"entity.mutantmore.mutant_shulker_trap" : "突变潜影贝陷阱",
+	"entity.mutantmore.mutant_blaze_rod_projectile" : "突变烈焰棒",
+	"entity.mutantmore.mutant_blaze_shields" : "突变烈焰人盾牌",
+	"entity.mutantmore.zombie_resurrection" : "复活的僵尸",
+	"entity.mutantmore.area_damage" : "吐息伤害",
+	"entity.mutantmore.sand_boulder" : "飞沙石",
+	"entity.mutantmore.icicle_spike" : "刺冰锥",
+	"entity.mutantmore.thrown_icicle" : "掷出的冰锥",
+	"entity.mutantmore.mutant_frozen_zombie" : "突变霜冻僵尸",
+	"entity.mutantmore.ice_cube" : "冰立方",
+	"entity.mutantmore.giant_snowball" : "巨型雪球",
+	"entity.mutantmore.mutant_jungle_zombie" : "突变丛林僵尸",
+	"entity.mutantmore.sentry_vine" : "哨戒藤",
+		
+	"item.mutantmore.mutant_wither_skeleton_spawn_egg" : "突变凋灵骷髅刷怪蛋",
+	"item.mutantmore.mutant_shulker_spawn_egg" : "突变潜影贝刷怪蛋",
+	"item.mutantmore.mutant_blaze_spawn_egg" : "突变烈焰人刷怪蛋",
+	"item.mutantmore.rodling_spawn_egg" : "烈焰灵刷怪蛋",
+	"item.mutantmore.mutant_husk_spawn_egg" : "突变尸壳刷怪蛋",
+	"item.mutantmore.mutant_hoglin_spawn_egg" : "突变疣猪兽刷怪蛋",
+	"item.mutantmore.mutant_frozen_zombie_spawn_egg" : "突变霜冻僵尸刷怪蛋",
+	"item.mutantmore.mutant_jungle_zombie_spawn_egg" : "突变丛林僵尸刷怪蛋",
+	"item.mutantmore.mutant_phantom_spawn_egg" : "突变幻翼刷怪蛋",
+	
+	"filled_map.derelict_laboratory" : "研究所探险家地图",
+	
+	"mutantmore.mutation_potion_complete" : "完成品",
+	"mutantmore.mutation_potion_progress" : "当前进展： %s/%s",
+	"mutantmore.mutation_potion_mutate_chance" : "突变成功率：%s",
+	"mutantmore.mutation_potion_unstable" : "不稳定成品",
+	"item.mutantmore.incomplete_formula_y.empty" : "空的化合物Y瓶",
+	"item.mutantmore.incomplete_formula_y.filled" : "满的化合物Y瓶",
+	"item.mutantmore.incomplete_formula_y" : "未完成的化合物Y",			
+	"item.mutantmore.formula_y" : "化合物Y",	
+	"item.mutantmore.incomplete_compound_z.empty" : "空的复合物Z瓶",
+	"item.mutantmore.incomplete_compound_z.filled" : "满的复合物Z瓶",
+	"item.mutantmore.incomplete_compound_z" : "未完成的复合物Z",
+	"item.mutantmore.compound_z" : "复合物Z",
+	
+	"mutantmore.adaptive_crossbow.desc" : "需放入装填弹才能发射",
```

| **审查级别**    |
|:-:|
| ❓疑问 / 需要作者确认 |

**原文: Loads and shoots Ammo Charges**

```suggestion
	"mutantmore.adaptive_crossbow.desc" : "需装填弹药才能发射",
```